timesketch/lib/analyzers/yetiindicators.py

```diff
@@ -91,41 +94,38 @@ def run(self):
         if not self.yeti_api_root or not self.yeti_api_key:
             return 'No Yeti configuration settings found, aborting.'
 
-        self.get_intrusion_sets()
-        actors_found = []
-        for intrusion_set in self.intel.values():
-            if not intrusion_set['indicators']:
-                continue
-
-            found = False
-
-            for indicator in intrusion_set['indicators']:
-                query = build_query_for_indicators([indicator])
-
-                events = self.event_stream(query_string=query,
-                                           return_fields=[])
-
-                name = intrusion_set['name']
-                for event in events:
-                    found = True
-                    event.add_emojis([emojis.get_emoji('SKULL')])
-                    event.add_tags([name])
-                    event.commit()
-                    event.add_comment(
-                        'Indicator "{0:s}" found for actor "{1:s}"'.format(
-                            indicator['name'], name))
-
-            if found:
-                actors_found.append(name)
-                self.sketch.add_view(
-                    'Domain activity for actor {0:s}'.format(name),
-                    self.NAME,
-                    query_string=query)
-
-        if actors_found:
-            return '{0:d} actors were found! [{1:s}]'.format(
-                len(actors_found), ', '.join(actors_found))
-        return 'No indicators were found in the timeline.'
+        self.get_indicators('x-regex')
+
+        entities_found = set()
+
+        events = self.event_stream(query_string='*',
```

Comment:
All events? Not even some sort of filter, on data_type maybe?

Reply:
I am happy to keep all events here if we're gonna use the generic regex approach. Since (for now) we don't really know "where" to look for a specific regex, it's probably better to pass it on all events. 

This would change if the Yeti object had specified a location where the regex is relevant via a tag or an extended field. But this is ontology work I'm not ready to dive in on this PR :)

Reply:
Ack, SG.